week7/answers.sql

```diff
@@ -1,3 +1,212 @@
--- Create your tables, views, functions and procedures here!
+-- Create your tables, views, functions, and procedures here!
 CREATE SCHEMA destruction;
 USE destruction;
+
+CREATE TABLE players (
+  player_id INT UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
+  first_name VARCHAR(30) NOT NULL,
+  last_name VARCHAR(30) NOT NULL,
+  email VARCHAR(50) NOT NULL
+);
+
+CREATE TABLE characters (
+  character_id INT UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
+  player_id INT UNSIGNED,
+  name VARCHAR(50) NOT NULL,
+  level INT,
+  FOREIGN KEY (player_id) REFERENCES players(player_id)
```

Comment:
Be sure to define your behavior (ON UPDATE, ON DELETE) for your FOREIGN KEY. I HIGHLY recommend naming you CONSTRAINTs, as well. (Apply this comment throughout the following code.)